tsl/src/chunk.c

```diff
@@ -632,6 +633,22 @@ merge_chunks_lock_upgrade_mode(void)
 	return MERGE_LOCK_ACCESS_EXCLUSIVE;
 }
 
+/*
+ * Use anonymous settings value to disable multidim merges due to a bug in the
+ * routing cache with non-aligned partitions/chunks.
+ */
+static bool
+merge_chunks_multidim_allowed(void)
+{
+	const char *multidim_allowed =
+		GetConfigOption("timescaledb.merge_chunks_multidim_allowed", true, false);
```

Comment:
Should we need to mention this anon GUC on the CHANGELOG?

Reply:
This is used only to make the tests compatible with the new block error. Not for user consumption.